.gitignore

```diff
@@ -16,6 +17,7 @@ buck-out
 **/BUCK
 
 buildSrc/bintray.properties
+buildSrc/secring.gpg
```

Comment:
Helps with local testing

Reply:
Nit: I don't think there should be a dev flow where there's a local secring. The signings creds should live in $HOME/.gradle which is outside the repo, and the secreing typically sits in a separate user path.

This is fine overall, as it's mostly a no-op. Just more curious on the flow that made this needed.